### ISSUE TITLE
Detect 10 digit American Phone Numbers

### ISSUE THREAD

**Issue by King-Jaden** [NCATComp410/comp410_fall_2023#6]

**Comment by claesmk** [comment]
You need to add a description

**Comment by King-Jaden** [comment]
> You need to add a description

Thanks you. I believe I just completed my description.

**Comment by claesmk** [comment]
Description looks good.  I would also add that a period is often used as a digit separator and should be include in your test cases - xxx.xxx.xxxx 

My recommendation would be to separate possible phone number detection from validation.  You could open a separate issue later if you decide you want to try to validate the area code on possible phone numbers.

**Comment by Ltchance** [comment]
Will this handle foreign phone numbers as well? They have a different starting digit based on their country.

**Comment by King-Jaden** [comment]
> Will this handle foreign phone numbers as well? They have a different starting digit based on their country.

No. Just for the sake of simplicity, I will only be handling and searching for American Phone Numbers, and assume unlabeled phone numbers (Those without the initial value signifying an international country code) are American.

**Comment by KevSmoot19** [comment]
I like how you covered all the various formats that phone numbers can abide by.

**Comment by claesmk** [comment]
> Will this handle foreign phone numbers as well? They have a different starting digit based on their country.

#34 will attempt to detect some foreign numbers

**Comment by claesmk** [comment]
@King-Jaden I think you will find that the default Presidio recognizer discussed in the sprint instructions will work pretty well for detecting phone numbers
